react-example/src/components/EventsForm.tsx

```diff
@@ -22,35 +29,61 @@ export const EventsForm: FC<Props> = ({
   heading,
   needsEventName
 }) => {
+  const { loggedInUser, setLoggedInUser } = useUser();
   const [trackResponse, setTrackResponse] = useState<string>(
     'Endpoint JSON goes here'
   );
 
-  const [trackEvent, setTrackEvent] = useState<string>('');
-
+  const eventInput =
+    endpointName === 'track'
+      ? '{"eventName":"button-clicked", "dataFields": {"browserVisit.website.domain":"https://mybrand.com/socks"}}'
+      : '';
```

Comment:
Why shouldn't this just be a JSON object instead?

Reply:
We have considered json string as default input value for code consistencty because user will also input the json string from in sample app and so we have to basically parse it to JSON Object in both the scenarios.

Reply:
Hmm, i think from the sample app, i'd also expect it to be a json object.

Reply:
@mprew97 Can you please explain how it can be a JSON object when any input we pass is considered a String?